packages/shopping/src/sequence.ts

```diff
@@ -33,12 +40,47 @@ export class MySequence implements SequenceHandler {
     try {
       const {request, response} = context;
       const route = this.findRoute(request);
-      await this.authenticateRequest(request);
+
+      //
+      // The authentication action utilizes a strategy resolver to find
```

Comment:
You can use a comment block instead but it's not a big deal so feel free to ignore